.circleci/cimodel/data/windows_build_definitions.py

```diff
@@ -108,7 +108,7 @@ def get_elements(self):
         return [self.prefixed_year()] + self.version_elements
 
     def get_product(self):
-        return "Community" if self.year == 2019 else "BuildTools"
+        return "BuildTools"
```

Comment:
Are BuildTools product line discontinued for VS2019?

Reply:
I only find enterprise, professional and buildtools link in
https://docs.microsoft.com/en-us/visualstudio/releases/2019/history#release-dates-and-build-numbers.
I'm afraid there might be license issue for enterprise and profession version.
So I have to choose BuildTools